app/ante/ante.go

```diff
@@ -7,90 +7,88 @@ import (
 		ante.NewMempoolFeeDecorator(),
 		ante.NewValidateBasicDecorator(),
-		ante.TxTimeoutHeightDecorator{},
-		ante.NewValidateMemoDecorator(ak),
-		ante.NewConsumeGasForTxSizeDecorator(ak),
-		ante.NewSetPubKeyDecorator(ak), // SetPubKeyDecorator must be called before all signature verification decorators
-		ante.NewValidateSigCountDecorator(ak),
-		ante.NewDeductFeeDecorator(ak, bk, fk),
-		ante.NewSigGasConsumeDecorator(ak, gasConsumer),
-		ante.NewSigVerificationDecorator(ak, signModeHandler),
-		ante.NewIncrementSequenceDecorator(ak),
-	)
+		ante.NewTxTimeoutHeightDecorator(),
+		ante.NewValidateMemoDecorator(options.AccountKeeper),
+		ante.NewConsumeGasForTxSizeDecorator(options.AccountKeeper),
+		ante.NewDeductFeeDecorator(options.AccountKeeper, options.BankKeeper, options.FeegrantKeeper),
+		ante.NewSetPubKeyDecorator(options.AccountKeeper), // SetPubKeyDecorator must be called before all signature verification decorators
+		ante.NewValidateSigCountDecorator(options.AccountKeeper),
+		ante.NewSigGasConsumeDecorator(options.AccountKeeper, sigGasConsumer),
+		ante.NewSigVerificationDecorator(options.AccountKeeper, options.SignModeHandler),
+		ante.NewIncrementSequenceDecorator(options.AccountKeeper),
+	), nil
+
 }
 
-// AdjustGasPriceDecorator is a custom decorator to reduce fee prices .
-type AdjustGasPriceDecorator struct {
+// ReduceGasPriceDecorator is a custom decorator to reduce fee prices .
+type ReduceGasPriceDecorator struct {
 }
 
-// NewAdjustGasPriceDecorator create a new instance of AdjustGasPriceDecorator
-func NewAdjustGasPriceDecorator() AdjustGasPriceDecorator {
-	return AdjustGasPriceDecorator{}
+// NewReduceGasPriceDecorator create a new instance of ReduceGasPriceDecorator
+func NewReduceGasPriceDecorator() ReduceGasPriceDecorator {
+	return ReduceGasPriceDecorator{}
 }
 
-// AnteHandle adjusts the gas price based on the tx type.
-func (r AdjustGasPriceDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, next sdk.AnteHandler) (newCtx sdk.Context, err error) {
+// AnteHandle reduces the gas price to a lower value which is hardcoded.The ReduceGasPriceDecorator should only be used for specific transaction types to lower the fee cost.
+func (r ReduceGasPriceDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, next sdk.AnteHandler) (newCtx sdk.Context, err error) {
 	msgs := tx.GetMsgs()
-	if len(msgs) == 1 && (strings.Contains(strings.ToLower(sdk.MsgTypeURL(msgs[0])), strings.ToLower(disptypes.MsgTypeCreateDistribution)) ||
-		strings.Contains(strings.ToLower(sdk.MsgTypeURL(msgs[0])), strings.ToLower(disptypes.MsgTypeRunDistribution))) {
 
-		minGasPrice := sdk.DecCoin{
-			Denom:  "rowan",
-			Amount: sdk.MustNewDecFromStr("0.00000005"),
-		}
-		if !minGasPrice.IsValid() {
-			return ctx, sdkerrors.Wrap(sdkerrors.ErrLogic, "invalid gas price")
-		}
-		ctx = ctx.WithMinGasPrices(sdk.NewDecCoins(minGasPrice))
-		return next(ctx, tx, simulate)
-	}
-	minFee := sdk.ZeroInt()
+	var found bool
```

Comment:
This is the old ante handler, develop has the handler changes and rewritten for 0.44